Documentation/api-reference.md

```diff
@@ -360,6 +520,33 @@ public final class ExperienceEvent {
       */
     public Builder setXdmSchema(final Map<String, Object> xdm) {...}
 
+    /**
+     * Override the default datastream identifier to send this event's data to a different datastream.
+		 *
+		 * When using {@link Edge#sendEvent}, this event is sent to the Experience Platform using the
+     * datastream identifier {@code datastreamIdOverride} instead of the default Experience Edge
+     * configuration ID set in the SDK Configuration key {@code edge.configId}.
+		 *
+		 * @param datastreamIdOverride Datastream identifier to override the default datastream identifier set in the Edge configuration
+		 * @return instance of current builder
+		 * @throws UnsupportedOperationException if this instance was already built
+     *
+     */
+    public Builder setDatastreamIdOverride(final String datastreamIdOverride) {...}
+
+		/**
+		 * Override the default datastream configuration to trigger different datastream behaviors than the
+     * default ones for this event.
+		 *
+		 * When using {@link Edge#sendEvent}, this event is sent to the Experience Platform along with the
+		 * datastream overrides defined in {@code datastreamConfigOverride}.
+		 *
+		 * @param datastreamConfigOverride Map defining datastream configuration overrides for this Experience Event
+		 * @return instance of current builder
+		 * @throws UnsupportedOperationException if this instance was already built
+		 */
+		public Builder setDatastreamConfigOverride(final Map<String, Object> datastreamConfigOverride) {...}
+
```

Comment:
Can you fix the indentation here? I think you'll need to use spaces instead of tabs.